alert-policies/argocd/ApplicationOutOfSync.yml

```
@@ -5,7 +5,7 @@ description: |+
 
 type: STATIC
 nrql:
-  query: "SELECT count(argocd_app_info) FROM Metric WHERE sync_status = 'OutOfSync' AND job = 'argocd-metrics'"
+  query: "SELECT count(argocd_app_info) FROM Metric WHERE sync_status = 'OutOfSync' WHERE instrumentation.name = 'remote-write'"
```

I think there might be an accidental change in the syntax here!
```suggestion
  query: "SELECT count(argocd_app_info) FROM Metric WHERE sync_status = 'OutOfSync' AND instrumentation.name = 'remote-write'"
```

---

@josephgregoryii Done. Thanks for pointing that out. 👍

---

No worries! Thanks for fixing that :)